scripts/review/diff.ts

```diff
@@ -0,0 +1,143 @@
+import util from 'node:util';
+import fs from 'node:fs/promises';
+import parseDiff from 'parse-diff';
+
+type Options = {
+  diffPatchFileName: string;
+  verbose: boolean;
+};
+
+enum HighestChangeType {
+  Initial,
+  Removal,
+  Addition,
+}
+
+function parseOptions(): Options {
+  const options = {
+    diffPatchFileName: { type: 'string' },
```

Comment:
uber-naming-nit: feels overly verbose
```suggestion
    patch: { type: 'string' },
```

PS: Leave as is if you disagree.

Reply:
Leaving it as it is 😁